treeherder/perf/models.py

```diff
@@ -649,3 +649,11 @@ class PerformanceSettings(models.Model):
 
     class Meta:
         db_table = "performance_settings"
+
+
+class PerformanceAlertTag(models.Model):
+    id = models.AutoField(primary_key=True)
+    name = models.CharField(max_length=30)
+
```

Comment:
We should also specify the many-to-many relation here.

Reply:
I updated the PR with the many-to-many relation.